jbpm-human-task/jbpm-human-task-jpa/src/main/java/org/jbpm/services/task/impl/model/ContentDataImpl.java

```diff
@@ -51,6 +51,16 @@ public void setType(String type) {
 	public void setContent(byte[] content) {
 		this.content = content;
 	}
+	
+	@Override
+	public Object getContentObject() {
+		throw new UnsupportedOperationException("Not implemented for JPA");
```

Comment:
wha would it take to implement this for JPA? bit uncomfortable to have new methods only for particular type ...